### PR TITLE
added notbanter

### PR DESCRIPTION
**Pull request by Shaqueel99**

### CODE REVIEW

**Comment by Shaqueel99** [comment]
hello can i pull pls babe

File: cogs/misc.py [hunk 1]
```diff
@@ -3,17 +3,26 @@
 import random
 import datetime
 import json
-from discord.ext import commands
+from discord.ext import commands, tasks
 from discord import app_commands
+import asyncio
+import pytube
```
Comment (audrickkena) [inline cogs/misc.py]: comment out

File: cogs/misc.py [hunk 2]
```diff
@@ -3,17 +3,26 @@
...
+
+
+
 
 class Misc(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
+        self.vc = None
```
Comment (audrickkena) [inline cogs/misc.py]: not needed, comment out

File: cogs/misc.py [hunk 3]
```diff
@@ -3,17 +3,26 @@
...
+        self.stop_audio_task = None
```
Comment (audrickkena) [inline cogs/misc.py]: not needed, comment out

File: cogs/misc.py [hunk 4]
```diff
@@ -30,7 +39,7 @@ async def rollDice(self, interaction: discord.Interaction, dice_num : int, sides
         if(sides_num > 30):
             await interaction.response.send_message('Sorry max value for the number of dice is 30!')
             return
-        if(interaction.channel.name=="command-spam"):
+        if(interaction.channel.name=="general"):
```
Comment (audrickkena) [inline cogs/misc.py]: remove as different interaction channel is used (command-spam)

File: cogs/misc.py [hunk 5]
```diff
@@ -73,11 +82,50 @@ def fileExists(self, filename):
         
     @app_commands.command(name='banter', description='For reminding everyone that what you said was just a joke')
     async def banter(self, interaction: discord.Interaction):
-        tts_channel = discord.utils.get(interaction.guild.text_channels, name='voiceless-spam-lvl10')
```
Comment (audrickkena) [inline cogs/misc.py]: revert back to this thanks

File: cogs/misc.py [hunk 6]
```diff
@@ -73,11 +82,50 @@ def fileExists(self, filename):
...
+        tts_channel = discord.utils.get(interaction.guild.text_channels, name='general')
```
Comment (audrickkena) [inline cogs/misc.py]: comment out or delete

File: cogs/misc.py [hunk 7]
```diff
@@ -30,7 +39,7 @@ async def rollDice(self, interaction: discord.Interaction, dice_num : int, sides
         if(sides_num > 30):
             await interaction.response.send_message('Sorry max value for the number of dice is 30!')
             return
-        if(interaction.channel.name=="command-spam"):
```
Comment (audrickkena) [inline cogs/misc.py]: revert to this

**Review by audrickkena — CHANGES_REQUESTED** [review]
Check comments for changes

**Review by audrickkena — APPROVED** [review]
Approved new command addition notbanter

**Pull request merged**